en/lessons/ecto/querying.md

```diff
@@ -189,33 +190,35 @@ iex> Repo.all(query)
 
 ### Getting the First and Last Records
 
-We can fetch the first or last records from a repository using the `Ecto.Query.first/2` and `Ecto.Query.last/2` functions.
+We can fetch the first or last records from a repository using the `Ecto.Query.first/1` and `Ecto.Query.last/1` functions.
```

Comment:
The `Ecto.Query.first` function does have an arity of two, see docs [here](https://hexdocs.pm/ecto/Ecto.Query.html#first/2). The second argument is optional and we're not using it in this example.

Reply:
So, shows the total number of arities, ok!

Reply:
```suggestion
We can fetch the first or last records from a repository using the `Ecto.Query.first/2` and `Ecto.Query.last/2` functions.
```